lib/rules/no-buffer-constructor.js

```diff
@@ -10,13 +10,17 @@
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
The `Buffer` constructor is deprecated in all currently-supported versions of Node.js. Do you think that is sufficient to consider this a problem?

Reply:
Good point, I agree.